docs/UserGuide.md

```diff
@@ -495,16 +482,16 @@ Note:
 
 ### Resetting hidden columns : `reset`
 
-Resets the columns in the table to the default view with *all* columns visible.
+Resets the columns in the table to the full view with *all* columns visible.
 
-Use this when you have called `showonly` or `hideonly` on the table!
+[comment]: <> (Use this when you have called `showonly` or `hideonly` on the table!)
 
 Format: `reset`
 
-Note:
 * Any input entered after the `reset` command will be ignored.
```

Comment:
This could potentially be misinterpreted as any input after using the `reset` command will be ignored. I recommend changing it to "Any input entered after the `reset` command word will be ignored." or "Any input entered directly after `reset` will be ignored".